fuel/datasets/text.py

```diff
@@ -98,6 +121,13 @@ def open(self):
         return chain(*[iter_(open_(f, encoding=self.encoding))
                        for f in self.files])
 
+    def _get_from_dictionary(self, symbol):
+        if symbol in self.dictionary:
```

Comment:
Nitpicking, but since this is potentially done many times, it's worth considering doing something like this, so that we only perform 1 dictionary lookup instead of 2.
```python
value = self.dictionary.get(symbol)
if value is None:
    return self.dictionary[self.unk_token]
```